docs/section6/Rule6-5.md

```diff
@@ -51,18 +51,19 @@
 
           - Get normalized space lighting schedule in P_RMR: `normalized_schedule_p = normalize_space_schedules(space_p.interior_lighting)`
 
-        - Check if automatic shutoff control is modeled in space during building closed hours (i.e. if lighting schedule hourly value in B_RMR is equal to P_RMR during building closed hours): `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_p, building_open_schedule_b, -111)`  
+        - Check if automatic shutoff control is modeled in space during building closed hours (i.e. if lighting schedule hourly value in B_RMR is equal to P_RMR during building closed hours): `schedule_comparison_result = compare_schedules(normalized_schedule_p, normalized_schedule_b, building_open_schedule_b, -111)`  
 
```

Comment:
I am not following what -111 means. I think it should be 1.

Also, in order to run compare_schedule function for hours when building is closed, the mask schedule should be an inverse of building_open_schedule, with value of 0 (exclude from comparison) where building_open_schedule_b=1 and value of 1 (include into comparison) where building_open_schedule_b=0.